nix/README.md

```diff
@@ -1,29 +1,35 @@
 ### Nix support
 
+### Quickstart
+
+- `sudo bash -c “mkdir /nix && chmod 755 /nix && chown -R $USERNAME /nix”`
+- Run the multi-user command from `https://nixos.org/download.html#nix-install-linux`
```

Comment:
Why the multi-user command?

Reply:
Instead of the single user command. Because my default inclination would be to install the one that requires fewer permissions. Also I don't want someone else running another daemon on my instance.

Reply:
Actually agree with this.